contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -15,6 +15,8 @@ import "./interfaces/KeeperRegistryInterface.sol";
 import "./interfaces/MigratableKeeperRegistryInterface.sol";
 import "./interfaces/UpkeepTranscoderInterface.sol";
 import "./interfaces/ERC677ReceiverInterface.sol";
+import "./interfaces/OptimismGasInterface.sol";
```

Comment:
can we include this package directly from op package such as https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/dev/OptimismSequencerUptimeFeed.sol#L11

Reply:
Done

Reply:
the optimism contract OVM_GasPriceOracle.sol requires 0.8.9 or above but we are on 0.8.6 so i stick with OptimismGasInterface